tests/filecheck/backend/riscv/convert_arith_to_riscv_snitch.mlir

```diff
@@ -0,0 +1,29 @@
+// RUN: xdsl-opt -p convert-arith-to-riscv-snitch,reconcile-unrealized-casts %s | filecheck %s
+
+// CHECK:   builtin.module
+// CHECK-NEXT:    %l, %r = "test.op"() : () -> (!riscv.freg, !riscv.freg)
```

Comment:
```suggestion
// CHECK:   builtin.module
// CHECK-NEXT:    %[[L.*]], %[[R:.*]] = "test.op"() : () -> (!riscv.freg, !riscv.freg)
```
Since https://github.com/xdslproject/xdsl/commit/9b43500e6526196b31b5a27a413af9475c1b71e0 landed, does this mean we can use named SSA variables now?

Reply:
Why do it in this case if the variable names are stable and more readable?

Reply:
I'll merge for now since you've approved but happy to refactor tests together tomorrow, there's more to come